src/controllers/controller.bar.js

```diff
@@ -453,8 +461,14 @@ module.exports = function(Chart) {
 
 			helpers.canvas.clipArea(chart.ctx, chart.chartArea);
 
+			//  float-bar support, if y arguments are array function will use bottom value as bar start point
 			for (; i < ilen; ++i) {
-				if (!isNaN(scale.getRightValue(dataset.data[i]))) {
+				var yValue = dataset.data[i];
+				if (helpers.isArray(yValue)) {
```

Comment:
Could we simplify this section by checking the parsed values instead of original data? This would mean we can remove an `if/else` statement. It also would mean the user can use non-array data (i.e. I don't have to override this method in the financial chart)

```
for (; i < ilen; ++i) {
  var val = scale._parseValue(dataset.data[i]);
  if (!helpers.isNullOrUndef(val.start) && !helpers.isNullOrUndef(val.end)) {
    rects[i].draw();
  }
}
```